### ISSUE TITLE
Order names / ID not consistent

### ISSUE THREAD

**Issue by Popolechien** [kiwix/cardshop#64]
Order confirmation number in the email is #5c6f3483fb0, but further down it is listed as 5c6f3483**98c8bf323d6aa**fb0 (ie with an extra 13 digits inserted in the middle), and the item ID on the cardshop is L25R5C6F.

**Comment by rgaudin** [comment]
Yes currently we have one (long) ID for the scheduler and a shorter one for the manager (supposedly more user friendly).

To skip a matching table, the short one is just a combination of characters from the long one. Hence short IDs are very common to each other and thus not useful IDs…

We need to find a better solution.

**Comment by Popolechien** [comment]
Why not use the item ID (``Lxyz``) all the way through? Or at least the same ID (long, short or item) throughout the process?
